Magick.NET.sln

```diff
@@ -1,7 +1,7 @@
 ﻿

```

Comment:
Can you revert your changes to this file? This should not be changed.

You can revert the file like this:
`git reset 0d330e17505053fa285d733f7cac50ff3dab1814 Magick.NET.sln`

And then commit it again.